app/graphql/resolvers/base_resolver.rb

```diff
@@ -42,8 +42,9 @@ def matching_user(submission, session_id)
       submitted_by_current_session?(submission, session_id)
   end
 
-  def matching_email(submission, user_email)
-    submission.user_email.downcase == user_email.downcase
+  def matching_email(submission)
+    user = Gravity.client.user(id: @context[:current_user]).user_detail._get
```

Comment:
Could you please add try...catch here

Reply:
I think line 11 in `add_user_to_submission_resolver` would catch it if anything goes wrong here. but I'll add a more specific exception handler 👍